src/sentry/api/endpoints/organization_user_details.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import OrganizationMember
+
+
+class OrganizationUserDetailsEndpoint(OrganizationEndpoint):
```

Comment:
I think we probably want to use:

https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_member_index.py#L25 

within `PermissionClasses` here – requiring the Integration to have selected the `member:read` permission at the least.

The way it is now, it only requires `org:read`, which I don't think makes sense for a users/members endpoint.